tools/windows-node-installer/pkg/resource/file.go

```diff
@@ -0,0 +1,99 @@
+package resource
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	set "github.com/deckarep/golang-set"
+)
+
+type InstallerInfo struct {
+	InstanceIDs      []string `json:"InstanceIDs"`
+	SecurityGroupIDs []string `json:"SecurityGroupIDs"`
+}
+
+const InstallerInfoFileName = "windows-node-installer.json"
```

Comment:
This constant need not be exported. I see you are exporting this for error messages and you shouldn't be doing that. The caller needn't know the name of the file and is internal to this package.